testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -2183,6 +2183,85 @@ public void testTrinoAnalyze()
         onTrino().executeQuery("DROP TABLE " + trinoTableName);
     }
 
+    @Test(dataProvider = "storageFormats")
+    public void testRegisterTableSuccess(StorageFormat storageFormat)
+    {
+        String baseOldTableName = "test_register_table_success_old_" + randomTableSuffix() + "_" + storageFormat.name().toLowerCase(ENGLISH);
+        String trinoOldTableName = trinoTableName(baseOldTableName);
+        String sparkOldTableName = sparkTableName(baseOldTableName);
+
+        onTrino().executeQuery(format("CREATE TABLE %s (a int, b varchar, c boolean) with (format = '%s', partitioning = ARRAY['a'])", trinoOldTableName, storageFormat));
+        onTrino().executeQuery(format("INSERT INTO %s values(1, 'INDIA', true)", trinoOldTableName));
+        onTrino().executeQuery(format("INSERT INTO %s values(2, 'USA', false)", trinoOldTableName));
+        onTrino().executeQuery(format("COMMENT ON TABLE %s IS '%s'", trinoOldTableName, "This is table comment"));
```

Comment:
Why do we have the `COMMENT` statements?
I don't see any assertions made for them. Are they relevant?

Reply:
I am checking for `SHOW CREATE TABLE` to verify if the table properties and comments are getting copied to the new table or not.

Here is check
`assertEquals(showCreateTrinoOldTable.replaceFirst(trinoOldTableName, trinoNewTableName), showCreateTrinoNewTable);
`